src/docs/product/error-monitoring/index.mdx

```diff
@@ -1,7 +1,29 @@
 ---
-title: Error Monitoring
+title: Issues
 sidebar_order: 1
-description: "Learn more about how the configuration of your SDK helps you resolve issues quickly in sentry.io."
+description: "Learn more about the Issues page and how you can use it to efficiently triage issues."
 ---
 
+The Issues page of [sentry.io](https://sentry.io) is where you can view, [triage](/product/error-monitoring/issue-details/), and resolve error events sent by your application. A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.
```

Comment:
I’d like us to make this an _extremely_ brief summary, like the Performance homepage description

Reply:
Lots of repetitive use of "sentry.io" here

Reply:
# Images

> The Issues homepage is where you can find important groups of errors in your app. Filter by event properties such as level, browser, device, the user that error impacted, unhandled errors, etc. You can then [triage them](link) or inspect the [issue details](link) to understand that problem in more depth.

<img width="1420" alt="CleanShot 2021-03-17 at 16 23 34@2x" src="https://user-images.githubusercontent.com/1900676/111551531-a5d6ba80-873d-11eb-8699-722ce321dd32.png">

### Issues are Groups of Error Events
A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, issues are grouped into similar events based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app; issues allow you to see how frequently that problem is happening and how many users it’s affecting. 

With this information, you can then make decisions about what to do next by [triaging the issue](link). For each issue, the page displays:

- Error type and message
- Brief description of the issue
- Associated project
- Issue timing, such as last and first time seen

### Organizing Issues 
The issues page is organized into tabs to help you group issues together:

- All Unresolved (is:unresolved): All unresolved issues, including issues in the "For Review" tab
- For Review (is:unresolved is:for_review): Issues that need to be reviewed
- Ignored (is:ignored): All ignored issues
- Saved Searches: Select from a set of premade and custom saved searches. The name of the tab changes based on your selection.

### Other Links
- Issue Details
- Using Breadcrumbs
- Issue States and Triage
- Issue Ownership Rules
- Reprocessing